### PR TITLE
add more rules

### LINKED ISSUE #4: Wrong singular form of "Taxes"
Am trying to use singular function to get the singular of the word "Taxes" but am getting "Taxe" instead of "Tax".

### PR DESCRIPTION
**Pull request by vixalien**
the rules were shamelessly stolen from [pluralize](https://github.com/plurals/pluralize).

Fix #4

### CODE REVIEW

**Comment by mauroerta** [comment]
> the rules were shamelessly stolen from [pluralize](https://github.com/plurals/pluralize).
> 
> Fix #4

Hi @vixalien, thank you a lot for your contribution!
Maybe it's better if we (at least) thank  [pluralize](https://github.com/plurals/pluralize) inside the README, can you please add a reference to pluralize inside the README?

Something like this at the bottom of the README:

## 🙏 Thanks

This project could not exist without the work of [pluralize](https://github.com/plurals/pluralize).

---
What do you think about it?

**Comment by vixalien** [comment]
Yeah that is the good thing to do.

Meanwhile, don't you think it would be cool if there was support for multiple languages?? I know the rules would be very hard to take hold of, but the availability of a way to customize the language would be a plus.

**Comment by vixalien** [comment]
But again, the project **COULD** exist without the work of pluralize. I'll just mention that the rules were got from the project.

**Comment by mauroerta** [comment]
> Yeah that is the good thing to do.
> 
> Meanwhile, don't you think it would be cool if there was support for multiple languages?? I know the rules would be very hard to take hold of, but the availability of a way to customize the language would be a plus.

For sure it could be cool, maybe we can open a discussion to understand what can be the right approach to do it

**Comment by mauroerta** [comment]
> But again, the project **COULD** exist without the work of pluralize. I'll just mention that the rules were got from the project.

Yes sure, but I'd like to properly thank them since the rules are the most important thing in this package and they are taken from them.

File: src/data.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const plurals: Record<string, string> = {
+const plurals_old: Record<string, string> = {
```
Comment (mauroerta) [inline src/data.ts:1]: Do you think we can remove this constant?

Comment (vixalien) [reply src/data.ts:1]: yessssss. I forgot

**Comment by vixalien** [comment]
> > But again, the project **COULD** exist without the work of pluralize. I'll just mention that the rules were got from the project.
> 
> Yes sure, but I'd like to properly thank them since the rules are the most important thing in this package and they are taken from them.

Nice what do you propose I put into the README then?

> For sure it could be cool, maybe we can open a discussion to understand what can be the right approach to do it

Yes. That's the right way to go

**Comment by mauroerta** [comment]
> > > But again, the project **COULD** exist without the work of pluralize. I'll just mention that the rules were got from the project.
> > 
> > 
> > Yes sure, but I'd like to properly thank them since the rules are the most important thing in this package and they are taken from them.
> 
> Nice what do you propose I put into the README then?

I think the following text at the end of the README:

## 🙏 Thanks
This project could not exist without the work of [pluralize](https://github.com/plurals/pluralize).

**Comment by vixalien** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -52,3 +52,7 @@ Testing plural --> singular
 The singular of feet is foot
 The singular of computers is computer
 ```
+
+## pray Thanks
```
Comment (mauroerta) [inline README.md:56]:
```suggestion
## 🙏 Thanks
```

Comment (mauroerta) [reply README.md:56]: If you can apply the suggestion above I'll merge, and again, thanks a lot for your contribution @vixalien

**Comment by vixalien** [comment]
fixed

**Comment by vixalien** [comment]
Yay!! would you care to start the previous discussion??

**Comment by vixalien** [comment]
Also, won't you create a new release/tag?? for this version to be pushed to deno.

**Comment by mauroerta** [comment]
> Also, won't you create a new release/tag?? for this version to be pushed to deno.

New version has been pushed to [deno](https://deno.land/x/deno_plural@1.1.0)

**Comment by vixalien** [comment]
Niceeeee

**Pull request merged**